src/main/java/wooteco/review/controller/GitHubController.java

```diff
@@ -52,10 +52,10 @@ public GitHubController(final GithubClientService githubClientService, final Rep
 		}
 
 		final GithubRepo githubRepo = GithubRepo.of(name);
-		final GithubRepo persistanceRepo = repoService.createRepo(githubRepo.withPullRequests(pullRequests));
+		final GithubRepo persistenceRepo = repoService.createRepo(githubRepo.withPullRequests(pullRequests));
 
 		return ResponseEntity
-			.created(URI.create("/repos/" + persistanceRepo.getId()))
+			.created(URI.create("/repos/" + persistenceRepo.getId()))
```

Comment:
제대로된 URI가 아닌거 같아요 ㅎㅎ
```suggestion
			.created(URI.create("api/github/repos/" + persistenceRepo.getId()))
```

Reply:
수정했습니다~!